dist/js/select2.js

```diff
@@ -4420,7 +4429,7 @@ S2.define('select2/dropdown/attachBody',[
 
     if (newDirection == 'above' ||
       (isCurrentlyAbove && newDirection !== 'below')) {
-      css.top = container.top - parentOffset.top - dropdown.height;
+      css.top = container.top - dropdown.height;
```

Comment:
I'm curious, why we had to make this change? and that too only for Select2 lib and not SelectWoo.

Reply:
This has been auto-compiled and bundled within here because of changes from a previous PR that didn't get compiled: https://github.com/woocommerce/selectWoo/pull/49/files